static/app/views/performance/data.tsx

```diff
@@ -551,9 +554,24 @@ function generateMobilePerformanceEventView(
     'p75(measurements.app_start_warm)',
     'p75(measurements.frames_slow_rate)',
     'p75(measurements.frames_frozen_rate)',
-    'p75(measurements.stall_percentage)',
   ];
 
+  // At this point, all projects are mobile projects.
+  // If in addition to that, all projects are react-native projects,
+  // then show the stall percentage as well.
+  const projectIds = genericEventView.project;
```

Comment:
Yeah this is fine for now, I think we can pull this out later when we do auto-display for mobile.